k8s/demo/common/probate/orchestrator-service.yaml

```diff
@@ -22,7 +22,7 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/probate/orchestrator-service:prod-e49f201c
+      image: hmctspublic.azurecr.io/probate/orchestrator-service:pr-322-latest
```

Comment:
can u please change it to-

 image: hmctspublic.azurecr.io/probate/orchestrator-service:latest